Updated files for child theme

Aelgiadi opened the pull request:
I updated the files and fixed the functions.php. If you see any problems or have any recommendations please let me know.

Aelgiadi committed:
Update functions.php

I ran the WP-DBUG and added ids to each register_sidebar. this caused it to no longer register an error.

Aelgiadi committed:
Delete body.php

since the spine is no longer sandwiched between the top and footer, there is no need for "spine-spacer" and therefore no need for this file.

Aelgiadi committed:
Update style.css

edited and updated the styles to fix some of the media queries and removed all styles for the spine.

jeremyfelt commented on functions.php:
```diff
@@ -1,6 +1,8 @@
+<?php	
+
  
 register_sidebar(array(
```
Any calls to `register_sidebar()` here should be wrapped inside a function that is attached to the `widgets_init` action. See https://github.com/washingtonstateuniversity/WSUWP-spine-parent-theme/blob/53b99e76c2f1049245dc4c5c198b48b2ffe1b93c/functions.php#L428-L442 as an example.

If these are not attached to an action, then the functions fire on every single page load, whether that page load supports side bars or not. Set `WP_DEBUG` to `true` in your local `wp-config.php` to see the errors this generates.

jeremyfelt commented:
Thanks for the updates @Aelgiadi, I left one comment on a commit for the sidebars. Once that's done, then we should be okay to push to staging.

Aelgiadi committed:
Made the necessary changes

@jeremyfelt I wrapped the register_sidebars() in a function, and attached it to widgets_init. i was a little confused, so please let me know if it was done correctly or not.

Aelgiadi committed:
Called all 3 dynamic_sidebars
